Packages/CommonUI/CommonUI/Classes/View/Button/UserGroupButton.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  UserGroupButton.swift
+//  CommonUI
+//
+//  Created by MinhDev on 17/06/2022.
+//
+
+import SwiftUI
+
+private enum Constants {
+	static let imageCache = URLCache(memoryCapacity: 512 * 1000 * 1000, diskCapacity: 10 * 1000 * 1000 * 1000)
+	static let cornerRadius = 4.0
+	static let padding = 3.0
+	static let lineWidth = 1.5
+	static let imageSize = CGSize(width: 60.0, height: 60.0)
+	static let spacing = 16.0
+}
+
+public struct UserGroupButton: View {
+	// MARK: - Variables
+
+	@Environment(\.colorScheme) var colorScheme
+	@Binding var title: String
+	private var imageUrl: URL?
+	private var action: (Bool) -> Void
+	private var isChecked: Bool = false
+	
+	// MARK: Init
+	public init(_ title: Binding<String>, imageUrl: String, isChecked: Bool, action: @escaping(Bool) -> Void) {
+		self._title = title
+		self.action = action
+        self.isChecked = isChecked
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e5e0a713c2a0f396e41d6374bb8f07759fda2936/Packages/CommonUI/CommonUI/Classes/View/Button/UserGroupButton.swift#L32"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>UserGroupButton.swift:32</code></td>
    </tr>
  </tbody>
</table>